pkgs/tools/security/browserpass/default.nix

```diff
@@ -32,7 +39,13 @@ buildGoModule rec {
   '';
 
   buildPhase = ''
-    make
+    make browserpass
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+    make test
+    runHook postCheck
```

Comment:
```suggestion
  buildFlags = [ "browserpass" ];
  checkTarget = "test";
```

Reply:
TIL, thanks! since browserpass uses `buildGoModule` instead of `mkDerivation`, `buildFlags` doesn't work as desired (gets passed to `go build` instead of `make`), but i applied the `checkTarget = "test"` and added you as co-author :)